### DISCUSSION TITLE
Env variables with array types

### DISCUSSION THREAD

**Discussion by mynavitechtus-minhnt3** [codemagic-ci-cd/codemagic-docs#1931] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I want to config `recipients` as an env variable but it does not work because `recipients` is an array type. How can I get there in CodeMagic? I wonder if there is any workaround here?
```    
artifacts:
  - app/app-develop-release.aab
publishing:
  email:
    recipients:
      - $EMAIL_RECIPIENTS
```

### Expected results

`emails` - `recipients` as an env variable

### Actual results

Not receive emails

**Comment by dtrdic** [comment]
hello @minhnt3, you can use anchors and aliases to define an array of recipients as an environment variable, something like this;

<img width="428" alt="image" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/c6190913-d48a-425e-8c5f-7d9a1a886e69">

**Reply by mynavitechtus-minhnt3** [reply]
@dtrdic Thank you but I want to hide the list of recipients from file `codemagic.yaml` by setting env variables on CodeMagic editor

**Reply by dtrdic** [reply, marked as answer]
Hey @minhnt3, unfortunately, it's not possible to use the environment variable on that field. However, I have forwarded your suggestion and will let you know if we make any changes

**Reply by chunglv-mynavitechtus** [reply]
@dtrdic 
Do you have any news?

**Reply by dtrdic** [reply]
hey @chunglv91 we decided to prioritise this feature request. I'll let you know once the feature is live
